plugins/languages/nvim-orgmode.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  helpers,
```

Comment:
Could you replace instances of `helpers` with `lib.nixvim` (our new name) instead.
```suggestion
```

Reply:
You may want to inherit `defaultNullOpts` to avoid those lines getting overly long:

```nix
let
  inherit (lib.nixvim) defaultNullOpts;
in
```